src/main/java/com/iridium/iridiumskyblock/commands/ExpelCommand.java

```diff
@@ -23,7 +23,7 @@
      * The default constructor.
      */
     public ExpelCommand() {
-        super(Collections.singletonList("expel"), "Kick a visitor", "%prefix% &7/is expel [<name>]", "", true, Duration.ZERO);
+        super(Collections.singletonList("expel"), "Kick a visitor", "%prefix% &7/is expel <player>", "", true, Duration.ZERO);
```

Comment:
idk about this but, i was did with [], its an optional arg so shouldn't be <> ?

Reply:
the syntax ive been using is () is optional <> is required

Reply:
No, you used <> everywhere, for example in all commands that have a GUI but an optional argument